packages/nextjs/components/project/ProjectHeader.tsx

```diff
@@ -58,14 +58,17 @@ const ProjectHeader = ({ project }: { project: ProjectDocument }) => {
 
   const addProjectToBallot = () => {
     const _name = project.name as string;
-    setNewAllocation(!Number.isNaN(newAllocation) && newAllocation > 0 ? newAllocation : 0);
+    // setNewAllocation(!Number.isNaN(newAllocation) && newAllocation > 0 ? newAllocation : 0);
+    const allocation = !Number.isNaN(newAllocation) && newAllocation > 0 ? newAllocation : 0;
+
+    const roundedAllocation = parseFloat(allocation.toFixed(2));
     dispatch({
       type: "ADD_PROJECT",
       project: {
         id: project._id,
         name: _name,
         category: project.category,
-        allocation: !Number.isNaN(newAllocation) && newAllocation > 0 ? newAllocation : 0,
+        allocation: roundedAllocation,
```

Comment:
It is fine that the displayed allocation uses toFixed to display the number cleanly but the actual allocation should be completely accurate.